src/pages/Form.tsx

```diff
@@ -0,0 +1,152 @@
+import style from "./Form.module.css";
+import Personal from "../components/Personal";
+import React, { Fragment } from "react";
+import { Button } from "@mui/material";
+import TravelInfo from "../components/TravelInfo";
+import TripInfo from "../components/TripInfo";
+import { styled } from "@mui/material/styles";
+import { Link } from "react-router-dom";
+
+const FormButton = styled(Button)(({ theme }) => ({
+  position: "fixed",
+  right: "-30px",
+  top: "50%",
+  backgroundColor: "#00da91",
+  fontSize: "1.5rem",
+  color: "black",
+  width: "14vw",
+  minWidth: "10rem",
+  padding: "1rem 1rem",
+  borderRadius: "5rem",
+  whiteSpace: "nowrap",
+  "&:hover": {
+    backgroundColor: "#116f59",
+    color: "#f0f0f0",
+  },
+  [theme.breakpoints.down("lg")]: {
+    fontSize: "1.2rem",
+  },
+  [theme.breakpoints.down("md")]: {
+    fontSize: "0.8rem",
+  },
+  [theme.breakpoints.down("sm")]: {
+    fontSize: "0.75rem",
+  },
+}));
+const HomeButton = styled(Button)(({ theme }) => ({
+  position: "fixed",
+  right: "-30px",
+  top: "42%",
+  backgroundColor: "#00da91",
+  fontSize: "1rem",
+  color: "black",
+  width: "12vw",
+  minWidth: "10rem",
+  padding: "1rem 1rem",
+  borderRadius: "5rem",
+  whiteSpace: "nowrap",
+  "&:hover": { backgroundColor: "#116f59", color: "#f0f0f0" },
+  [theme.breakpoints.down("lg")]: {
+    fontSize: "1.2rem",
+  },
+  [theme.breakpoints.down("md")]: {
+    fontSize: "0.8rem",
+  },
+  [theme.breakpoints.down("sm")]: {
+    fontSize: "0.75rem",
+  },
+}));
+const ResetButton = styled(Button)(({ theme }) => ({
```

Comment:
Can this be a separate component? Maybe you can try to combine different buttons in one component and change its look by using some props?